src/dispatch/config.py

```diff
@@ -44,19 +44,24 @@ def __str__(self) -> str:
 DISPATCH_HELP_SLACK_CHANNEL = config("DISPATCH_HELP_SLACK_CHANNEL")
 
 # authentication
-DISPATCH_PKCE_AUTH_ENABLED = config("DISPATCH_PKCE_AUTH", cast=bool, default=True)
-DISPATCH_HEADER_AUTH_ENABLED = config("DISPATCH_HEADER_AUTH", cast=bool, default=False)
-
-DISPATCH_AUTH_HEADER_KEY = config("DISPATCH_AUTH_HEADER", default=None)
-JWKS_URL = config("JWKS_URL", default=None)
+DISPATCH_AUTHENTICATION_PROVIDER_SLUG = config(
+    "DISPATCH_AUTHENTICATION_PROVIDER_SLUG", default="dispatch-auth-provider-pkce"
+)
+VUE_APP_DISPATCH_AUTHENTICATION_PROVIDER_SLUG = DISPATCH_AUTHENTICATION_PROVIDER_SLUG
```

Comment:
Should we get rid of `VUE_APP_DISPATCH_AUTHENTICATION_PROVIDER_SLUG` if we're always setting it to `DISPATCH_AUTHENTICATION_PROVIDER_SLUG`, and use `DISPATCH_AUTHENTICATION_PROVIDER_SLUG` instead?

Reply:
Unfortunately, Vue will only pick up envvars prefixed with `VUE_APP` to be injected into the vue app. I think this is the best of both worlds (only having to set one var).